releases/nightly-build/compose-files/Makefile

```diff
@@ -0,0 +1,129 @@
+.PHONY: help portainer portainer-down run pull gen down clean remove
+.SILENT: help get-token del-token
+
+# TODO: When cutting next release change these to edgexfoundry & the actual Edgex release version
+REPOSITORY:=nexus3.edgexfoundry.org:10004
+CORE_EDGEX_VERSION:=master
+# Note: Device services, consul and App Services versions may differ from the Edgex release version
+#	    The versions for these services still have to be changed in the compose file as they all may be different.
+
+COMPOSE_FILES:=-f docker-compose-nexus-base.yml
+OPTIONS:=" arm64 no-secty no-ds mqtt dev ui " # Must have spaces around words for `filter-out` function to work properly
+
+ARGS:=$(wordlist 2,$(words $(MAKECMDGOALS)),$(MAKECMDGOALS))
+$(eval $(ARGS):;@:)
+
+ifeq (dev, $(filter dev,$(ARGS)))
+	DEV:=-dev
+endif
+ifeq (arm64, $(filter arm64,$(ARGS)))
+	ARCH:=-arm64
+	REDIS_ARCH:=arm64v8/
```

Comment:
@hutchic , are you saying that the `redis:5.0.8-alpine` image works for x86 as well as ARM?

Reply:
Yup exactly 👍 

![image](https://user-images.githubusercontent.com/697188/85031193-63e7fc00-b14c-11ea-853b-d226fa2a7229.png)

Docker will pull / run the correct image that matches your architecture one shouldn't need to pin the architecture

Reply:
Interesting. @andresrinivasan , is there another reason you originally choose `arm64v8/redis:5.0.8-alpine` for the ARM compose files rather than the `redis:5.0.8-alpine`?

Reply:
Good catch. Previously, the standard image did not include ARM64. Very happy to see we're including that architecture now.

Reply:
Ok then... switching to just the one image. THX @hutchic for catching this!